lib/Scvmm/miq_scvmm_vm_ssa_info.rb

```diff
@@ -0,0 +1,69 @@
+$LOAD_PATH.push("#{File.dirname(__FILE__)}/../util")
```

Comment:
which of the requires below is coming from util?  if none, then we do not need this push.